### DISCUSSION TITLE
focus

### DISCUSSION THREAD

**Discussion by ddkwork** [cogentcore/core#736] [General]
I'm not sure if the problem can be properly described by a translation engine: the focus of all drop-down menu controls like combox seems to be a bit uncontrollable, and sometimes the focus is unresponsive, such as pulling down to the first member twice, and a possible fix is to actively request focus when the mouse hovers over the pulled member. This problem feels this way in all the controls like tables, trees, dropdowns, editors, etc. Generally speaking, it is the focus, the cursor and the mouse pass through and other behaviors are inconsistent, and there is a slight lag when that happens, maybe somewhere you need a callback to prompt real-time?

**Comment by kkoreilly** [comment]
I will look into this.
